dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tei/TeiQueryController.java

```diff
@@ -136,6 +136,7 @@ Grid getGrid(
     @GetMapping( "/query/dimensions" )
     public AnalyticsDimensionsPagingWrapper<ObjectNode> getQueryDimensions(
         @RequestParam String trackedEntityType,
+        @RequestParam( required = false ) List<String> program,
```

Comment:
I would use a Set, as the program cannot be repeated.

Reply:
done